cdn_lambda/functions/map_to_s3/map_to_s3.py

```diff
@@ -1,38 +1,42 @@
 import json
 import logging
-import os
 from datetime import datetime, timezone
 
 import boto3
 
 LOG = logging.getLogger("map-to-s3-lambda")
 
 
-def lambda_handler(event, context):
+def load_conf(conf_file):
+    with open(conf_file, "r") as json_file:
+        content = json.load(json_file)
+
+    return content
+
+
+def lambda_handler(event, context, conf_file=None):
```

Comment:
conf_file parameter is really only for testing.
The json file name needs to be hard-coded here to work in AWS but auto-tests can't locate it.
This seems the only reasonable way to enable testing without mocking open().